### PR TITLE
Simplify Payment Page

### LINKED ISSUE #1868: Simplify Payment Page: Move ticket holder person box and add box to enable billing information
On the payment page the following changes should be made (see image below. It should be like that.):
* move the ticket holder box to ask whether the ticket holder is the same person as the buys just below the heading
* below the heading "billing information" add a box "Enable Billing Information" 
   * standard should be disabled and in that case don't show the billing info fileds
   * if enabled show the fields to fill in additional data

screenshot from 2019-01-17 02-15-40

### PR DESCRIPTION
**Pull request by sk9331657**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

screenshot from 2019-01-20 16-29-40
screenshot from 2019-01-20 16-29-48
screenshot from 2019-01-20 16-29-59

#### Changes proposed in this pull request:

- Simplify Payment Page

Fixes #1868

### CODE REVIEW

**Comment by sk9331657** [comment]
@CosmicCoder96 @iamareebjamal  @mariobehling  Please review.

**Comment by sk9331657** [comment]
@niranjan94 Please review.

**Comment by iamareebjamal** [comment]
Please don't request reviews until it has been a day since no one has reviewed and the build is not faling

File: app/components/forms/orders/order-form.js [hunk 1]
```diff
@@ -40,7 +40,8 @@ export default Component.extend(FormMixin, {
     }
     return true;
   }),
-  sameAsBuyer: false,
+  sameAsBuyer   : false,
+  enablebilling : false,
```
[the remark this answers is not in the document]
Comment (sk9331657) [reply app/components/forms/orders/order-form.js]: Ok Will do that.

**Comment by sk9331657** [comment]
@CosmicCoder96  Please review .

**Comment by sk9331657** [comment]
@CosmicCoder96  Please review this.

**Comment by sk9331657** [comment]
@CosmicCoder96 @iamareebjamal @mariobehling  Please review.

**Pull request merged**